src/server/pachyderm_test.go

```diff
@@ -3183,11 +3184,22 @@ func TestStandby(t *testing.T) {
 		require.NoError(t, err)
 		commitInfos, err := c.WaitCommitSetAll(commitInfo.Commit.ID)
 		require.NoError(t, err)
-		require.Equal(t, 2, len(commitInfos))
+		nonAliasCommits := 0
+		for _, v := range commitInfos {
+			if v.Origin.Kind != pfs.OriginKind_ALIAS {
+				nonAliasCommits++
+			}
+		}
+		require.Equal(t, 3, nonAliasCommits) // we got an extra meta commit here
```

Comment:
The comment here is probably unnecessary unless you want to just list all three commits - otherwise this is just an explanation of the change from the previous version which is no longer present in the code to give the necessary context.

Reply:
good point. Will remove this! I also just wanted you to see this since you're aware of any more "syntactic sugar" like changes that may come in to deal with explore the different commit types